packages/rOpenScPCA/tests/testthat/test-calculate-clusters.R

```diff
@@ -1,9 +1,24 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+suppressPackageStartupMessages(library(Seurat))
+
+sce <- scpcaTools:::sim_sce(n_cells = 100, n_genes = 10, n_empty = 0)
```

Comment:
Not a big fan of using an internal function from a different package, even in testing. Maybe use `splatter::simpleSimulate()`?